src/components/Match/Vision/VisionFilter.jsx

```diff
@@ -19,30 +19,30 @@ const data = [
   },
 ];
 
-export default class VisionFilter extends React.PureComponent {
-  handleCheckGenerator(player, type) {
-    const parent = this.props.parent;
-
-    return event => parent.setPlayer(player, type, event.target.checked);
-  }
-
-  handleCheckGeneratorTeam(index) {
-    const parent = this.props.parent;
-
-    return event => parent.setTeam(index === 0 ? 'radiant' : 'dire', event.target.checked);
-  }
```

Comment:
These functions don't seem to be reused, and I find the new version more readable.  Is there a reason to have them defined separately and returning functions?

Behavior is the same.